src/components/flat-table/flat-table-row/flat-table-row.style.ts

```diff
@@ -278,7 +280,9 @@ const StyledFlatTableRow = styled.tr<StyledFlatTableRowProps>`
           }
 
           /* Styling for safari. Position relative does not work on tr elements on Safari  */
-          ${isSafari(navigator) &&
+          // FIXME: this can cause hydration mismatches during SSR.
```

Comment:
comment: we should probably add a JIRA ticket or Github Issue to this so that it doesn't get missed